Shniak -- Component Tests

Warumo opened the pull request:
This PR adds unit tests to all my components (MainContainer, TaskColumn,  TaskItem) and also change some structuring in my folders for test purposes. Several functions that are used in my components are now separated in `utils.js` files. Added comma in `babel.config.js` file according to the ESLint rules.

opanki commented on src/components/Dashboard/MainContainer/utils.js:
```diff
@@ -0,0 +1,17 @@
+  return taskList;
+};
+
+export default getTaskListAsArray;
```
no need to export getTaskListAsArray as default here

opanki commented on src/components/Dashboard/TaskItem/utils.js:
```diff
@@ -0,0 +1,23 @@
+  let style = status === 'Done' ? 'taskItemDone' : 'taskItem';
+  switch (priority) {
+    case 'High':
+      style += ' priorityH';
```
to avoid switch-case construction please use template strings here

opanki commented on src/components/Dashboard/TaskItem/utils.js:
```diff
@@ -0,0 +1,23 @@
+
+export const getTaskStyleByPriority = (props) => {
+  const { priority, status } = props;
+  let style = status === 'Done' ? 'taskItemDone' : 'taskItem';
```
it's common to use camelCase for naming variables, but kebab-case for css classes

camelCase
snake_case
kebab-case

FYI: https://medium.freecodecamp.org/css-naming-conventions-that-will-save-you-hours-of-debugging-35cea737d849

Warumo commented:
New commits change class names (now they are in kebab-case). I've also added new ESLint rule to disable a default exporting errors because that was needed to correct my utils files according to TL request. I've also changed our Readme file because it was a very unrelated to what we are doing, but we will improve it in the feature in any case.
